replica/database.cc

```diff
@@ -362,6 +363,15 @@ database::database(const db::config& cfg, database_config dbcfg, service::migrat
             utils::updateable_value(std::numeric_limits<uint32_t>::max()),
             utils::updateable_value(std::numeric_limits<uint32_t>::max()),
             reader_concurrency_semaphore::register_metrics::yes)
+    , _view_update_read_concurrency_sem(
+            utils::updateable_value<int>(max_count_concurrent_reads),
+            max_memory_concurrent_view_update_reads(),
+            "view_update",
+            max_inactive_view_update_queue_length(),
+            _cfg.reader_concurrency_semaphore_serialize_limit_multiplier,
+            _cfg.reader_concurrency_semaphore_kill_limit_multiplier,
+            _cfg.reader_concurrency_semaphore_cpu_concurrency,
```

Comment:
For these too.

Reply:
I added the dedicated config options in the rebase, for now their values are the same as for the user read semaphore.